scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -15,74 +15,98 @@
                     ng-if="can_unlock()"
-                    ng-click="unlock()" 
+                    ng-click="unlock()"
                     ng-disabled="unlockClicked" translate>
                 Unlock
             </button>
         </div>
     </div>
 
-    <div class="authoring-topbar clearfix pull-left" ng-class="{'has-highlights': item.highlights}">
+    <div class="subnav__authoring-topbar" ng-class="{'has-highlights': item.highlights}">
         <span class="stage" ng-show="stage" ng-if="item._type !== 'legal_archive'"><b>{{deskName}}</b> / {{ stage.name }}</span>
         <span class="stage" ng-show="stage" ng-if="item._type === 'legal_archive'"><b>{{deskName}}</b> / {{ stage }}</span>
         <div ng-if="item.more_coming === true" class="state-label state-in_progress" translate>More coming</div>
         <span ng-if="item.highlights" sd-highlights-title data-item="item"></span>
     </div>
 
-    <div class="button-stack right-stack">
-
-        <button id="Edit" class="btn btn-medium pull-left" ng-if="item._type !== 'legal_archive'"
-            ng-show="action === 'view' && !item._locked && itemActions.edit"
-            ng-click="edit()"
-            translate>Edit</button>
-
-        <button class="btn btn-medium pull-left"
-            ng-show="action === 'view' && !item._locked && itemActions.deschedule"
-            ng-click="deschedule()"
-            translate>Deschedule</button>
-
-        <button class="btn btn-medium btn-info pull-left" ng-if="item._type !== 'legal_archive'"
-            ng-show="action === 'view' && !item._locked && itemActions.correct"
-            ng-click="openAction('correct')"
-            translate>Edit and Correct</button>
-
-        <button class="btn btn-medium btn-info pull-left" ng-if="item._type !== 'legal_archive'"
-            ng-show="action === 'view' && !item._locked && itemActions.kill"
-            ng-click="openAction('kill')"
-            translate>Edit and Kill</button>
-
-        <button class="btn btn-medium pull-left uppercase" id="closeAuthoringBtn" ng-click="close()" translate>Close</button>
-
-        <button class="btn btn-medium btn-info pull-left uppercase" type="submit"
-            ng-click="saveTopbar(item)"
-            ng-show="_editable"
-            ng-if = "itemActions.save && action === 'edit'"
-            ng-disabled="!save_enabled() || saveDisabled"
-            translate>Save</button>
-
-        <button id="export" class="btn btn-medium btn-info pull-left"
-			 ng-click="exportHighlight(item)"
-			 ng-disabled="!_editable"
-			 ng-if="highlight"
-			 translate>Export</button>
-
-        <button class="navbtn navbtn-hover strict"
-                ng-if="!item._locked && _editable"
-                title="{{ :: 'Minimize' | translate }}"
-                ng-click="minimize()">
+    <div class="subnav__stretch-bar">
+      <div class="subnav__button-stack subnav__button-stack--custom-buttons" ng-show="_editable">
+        <button class="btn btn-custom"
```

Comment:
You would need the some rules for the these buttons. `_editable` is not sufficient. 
- the user can either `correct` or `kill` the item after publishing.
- the user can only deschedule the item if the item is scheduled.
- user can only send content to a different stage if the content is not published.

Check the `itemActions` method authoring service.